### ISSUE TITLE
Width seems to be wrong with custom none_format

### ISSUE THREAD

**Issue by OlafvdSpek** [jazzband/prettytable#172]
```python
from prettytable import PrettyTable
t = PrettyTable()
t.add_row([None])
t.none_format='0123456789'
print(t)
t.header = False
print()
print(t)
```
```
+---------+
| Field 1 |
+---------+
| 0123456 |
|   789   |
+---------+

+------+
| 0123 |
| 4567 |
|  89  |
+------+
```

**Comment by hugovk** [comment]
### What did you expect to happen? 

### What versions are you using?

* OS: 
* Python: 
* PrettyTable:

**Comment by OlafvdSpek** [comment]
Expected:
```
+------------+
|  Field 1   |
+------------+
| 0123456789 |
+------------+

+------------+
| 0123456789 |
+------------+
```
OS: Windows 10 and Debian unstable
Python 3.10 and 3.9
PrettyTable: git and 3.1.1

**Comment by hugovk** [comment]
Thanks!

And for comparison, adding a single row with the string `'0123456789'` also produces single-line output.

```python
from prettytable import PrettyTable
t = PrettyTable()
# t.add_row([None])
# t.none_format='0123456789'
t.add_row(['0123456789'])
print(t)
t.header = False
print()
print(t)
```
```
+------------+
|  Field 1   |
+------------+
| 0123456789 |
+------------+

+------------+
| 0123456789 |
+------------+
```

`none_format` was recently added in https://github.com/jazzband/prettytable/pull/164.

@av-guy Would you like to take a look at this? Thanks!

**Comment by av-guy** [comment]
Yes I will review immediately and see what's going on.

**Comment by av-guy** [comment]
@hugovk I was able to figure out what is causing this issue, and I will be making a PR for the fix later this evening.

**Comment by hugovk** [comment]
Thanks both, fixed in #174.
